VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.html

```diff
@@ -136,52 +134,45 @@ <h1 class="govuk-heading-xl govuk-!-padding-top-5">Your video hearing {{ getConf
     <div *ngIf="currentTime" class="govuk-panel" [ngClass]="getCurrentTimeClass()">
       <div class="govuk-body govuk-!-margin-bottom-0" [ngClass]="getCurrentTimeClass()">
         <div *ngIf="!isWitness && hearing.isOnTime()">
-          <h2 class="govuk-heading-m white">Please wait</h2>
-          <p class="white">Your video hearing is scheduled to begin at {{ hearing.scheduledStartTime | date: 'HH:mm'
-            }}
+          <h2 class="govuk-heading-m white">{{'participant-waiting-room.please-wait' | translate}}</h2>
+          <p class="white">{{'participant-waiting-room.hearing-starting-at' | translate}} {{ hearing.scheduledStartTime | date: 'HH:mm' }}
           </p>
         </div>
 
         <div *ngIf="!isWitness && hearing.isStarting()">
-          <h2 class="govuk-heading-m white">Please stay near your screen</h2>
-          <p class="white">Make sure you're ready when the judge starts the hearing.</p>
+          <h2 class="govuk-heading-m white">{{'participant-waiting-room.hearing-status-messages.starting' | translate}}</h2>
```

Comment:
title and body instead of starting and starting1 might be more descriptive?

Reply:
same for delayed

Reply:
Done